src/styles.css

```diff
@@ -19,11 +53,10 @@ body {
 .navbar {
```

Comment:
Try to maintain separate CSS files for each component and common attributes like padding, margin in styles.css. Maintaining a separate CSS will be useful for debugging.

Reply:
Thanks..! @vikbhas for the review and suggestions 
add wishlist button on productDetail page, add starting range value, seprate css for components, maintained uniformity of units.